index.html

```diff
@@ -74,22 +77,18 @@
             </div>
             <svg class="country-table" style="display: none"></svg>
             <div class="country-table-initial-text">
-                <span style="font-size: 1.3em; color: lightgray">This is the <b>electricity map</b></span><br />
-                <br />
-                It shows, in real-time, <span style="color: lightgray">where your electricity comes from</span>
-                and <span style="color: lightgray">how much greenhouse gas (GHG)</span> was emitted in the process.<br />
+                The map on the right shows in real-time <span style="color: lightgray">where your electricity comes from</span>
+                and <span style="color: lightgray">how much greenhouse gas (GHG)</span> was emitted to produce it.
                 <!--The map also shows renewable resources like wind and sun.<br />-->
                 The <span style="color: lightgray">blinking arrows</span> represent electricity <span style="color: lightgray">imports and exports</span>.<br />
                 <br />
-                &nbsp;&nbsp;&nbsp;&gt;&gt;&nbsp;&nbsp;<span style="color: lightgray">click on a country to get started</span><br />
-                <br />
-                Going further:<br />
-                &gt; <a href="https://github.com/corradio/electricitymap#greenhouse-gas-footprint-calcuation-and-data-source" target="_blank">Understand</a> how GHG emissions are calculated<br />
-                &gt; <a href="https://github.com/corradio/electricitymap#contribute" target="_blank">Contribute</a> to complete the map
+                <span style="color: lightgray; font-size: 14px; color: white;"><i>Tip: Click on a country to start exploring ⟶</i></span><br />
             </div>
+            <small class="contribute-text">To learn more about this project or contribute missing data <a href="https://github.com/corradio/electricitymap#contribute" target="_blank">head over to GitHub</a>.</small>
+        </div>
         </div>
-        <div class="contribute-bar">
```

Comment:
Can we try to keep it? I think it's necessary to show that it's open source (link to github) and to link to twitter so we get some audience. Maybe we can have both this and the watermark?